angular/core/components/poll/common/outcome_panel/poll_common_outcome_panel.haml

```diff
@@ -1,9 +1,9 @@
-.poll-common-outcome-panel
-  %md-list.md-block{ng-if: "!poll.outcome().statement"}
-    %h3.lmo-card-subheading{translate: "poll_common.set_outcome_title"}
-    %p.poll-common-helptext{translate: "poll_common.set_outcome_text"}
-    %md-button.md-primary{ng-click: "openOutcomeForm()", translate: "poll_common.set_outcome", aria-label: "{{poll_common.set_outcome | translate}}"}
-  %md-list.md-block{ng-if: "poll.outcome().statement"}
-    %h3.lmo-card-subheading{translate: "poll_common.outcome"}
-    %p {{poll.outcome().statement}}
-    %md-button.md-primary{ng-click: "openOutcomeForm()", translate: "poll_common.update_outcome", aria-label: "{{poll_common.update_outcome | translate}}"}
+.poll-common-outcome-panel{ng-if: "poll.outcome()"}
```

Comment:
I've run into issues in the past having ng-if on root elements of directives. Have we tested this while false?